tests/sentry/issues/test_escalating_issues_alg.py

```diff
@@ -0,0 +1,696 @@
+    data = {"intervals": SEVEN_DAY_INPUT_INTERVALS, "data": SEVEN_DAY_ERROR_EVENTS}
+
+    ceilings_list = issue_spike(data, start_time)
+    ceilings = [x["forecasted_value"] for x in ceilings_list]
+
+    assert ceilings == [6987] * 14, "Ceilings are incorrect"
+
+
+def test_bursty_case() -> None:
+    error_events = [
+        77,
+        162,
+        7846,
+        8,
+        4,
+        2,
+        3,
+        25,
+        4,
+        24,
+        7,
+        6,
+        3,
+        8,
+        3,
+        157,
+        168,
+        161,
+        7955,
+        1,
+        10,
+        19,
+        108,
+        155,
+        9,
+        1,
+        166,
+        152,
+        149,
+        6604,
+        2,
+        6,
+        198,
+        127,
+        182,
+        4032,
+        1,
+        6,
+        4,
+        198,
+        120,
+        171,
+        3656,
+        6,
+        4,
+        1,
+        199,
+        150,
+        159,
+        7113,
+        1,
+        10,
+        3,
+        2,
+        4,
+        2,
+        176,
+        146,
+        174,
+        8290,
+        6,
+        2,
+        1,
+        1,
+        9,
+        2,
+        6,
+        2,
+        2,
+        174,
+        144,
+        154,
+        7627,
+    ] + [0] * 95
+
+    data = {"intervals": SEVEN_DAY_INPUT_INTERVALS, "data": error_events}
+
+    ceilings_list = issue_spike(data, START_TIME)
+    ceilings = [x["forecasted_value"] for x in ceilings_list]
+
+    assert ceilings == [16580] * 14, "Ceilings are incorrect"
+
+
+def test_empty_input() -> None:
+    error_events: List[int] = []
+
+    data = {"intervals": SEVEN_DAY_INPUT_INTERVALS, "data": error_events}
+
+    ceilings_list = issue_spike(data, START_TIME)
+    ceilings = [x["forecasted_value"] for x in ceilings_list]
+
+    assert ceilings == [], "Empty Input"
+
+
+def test_less_than_week_data() -> None:
+    error_events = [
+        9,
+        1,
+        166,
+        152,
+        149,
+        6604,
+        2,
+        6,
+        198,
+        127,
+        182,
+        4032,
+        1,
+        6,
+        4,
+        198,
+        120,
+        171,
+        3656,
+        6,
+        4,
+        1,
+        199,
+        150,
+        159,
+        7113,
+        1,
+        10,
+        3,
+        2,
+        4,
+        2,
+        176,
+        146,
+        174,
+        8290,
+        6,
+        2,
+        1,
+        1,
+        9,
+        2,
+        6,
+        2,
+        2,
+        174,
+        144,
+        154,
+        7627,
+    ] + [0] * 95
+
+    data = {"intervals": SIX_DAY_INPUT_INTERVALS, "data": error_events}
+
+    ceilings_list = issue_spike(data, START_TIME)
+    ceilings = [x["forecasted_value"] for x in ceilings_list]
+
+    assert ceilings == [82900] * 14, "Ceilings are incorrect"
+
+
+def test_low_freq_events() -> None:
+    error_events = [6] * 168
+
+    data = {"intervals": SIX_DAY_INPUT_INTERVALS, "data": error_events}
+
```

Comment:
this should be changed to `SEVEN_DAY_INPUT_INTERVALS` -> its still 7 days worth of data just low # of events hourly so uses the floor bound instead